### ISSUE TITLE
Undefined function mb_str_split

### ISSUE THREAD

**Issue by valtzu** [web-token/jwt-framework#215]
**Describe the bug**
Starting from version 2.0.11, the RSA encryption fails with error of `Error: Call to undefined function Jose\Component\Encryption\Algorithm\KeyEncryption\Util\mb_str_split()`
I'm not sure if there is more mb_str_split calls in the project.

**To Reproduce**
1. Use JWEBuilder to build RSA encrypted JWE

**Expected behavior**
No error, like 2.0.10 works

**Additional context**
It is no wonder that this fails as mb_str_split does not exist until PHP 7.4. However composer.json says this library should work with PHP 7.2 also.

**Comment by Spomky** [comment]
Hi @valtzu,

You are right it looks like this method is only supported on PHP7.4.
php-cs-fixer modified it. I will revert that change asap.

**Comment by Spomky** [comment]
For the record: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4301#issuecomment-527968055

**Comment by Spomky** [comment]
Fixed in `v2.0.13`

**Comment by PlusA2M** [comment]
> Fixed in `v2.0.13`

@Spomky Same things happened on the latest version.

```
Fatal error: Uncaught Error: Call to undefined function Jose\Component\Encryption\Algorithm\KeyEncryption\Util\mb_str_split() in 
/path-to-www/vendor/web-token/jwt-encryption-algorithm-rsa/Util/RSACrypt.php:123 
Stack trace: 
#0 /path-to-www/vendor/web-token/jwt-encryption-algorithm-rsa/Util/RSACrypt.php(45): Jose\Component\Encryption\Algorithm\KeyEncryption\Util\RSACrypt::encryptWithRSAOAEP(Object(Jose\Component\Core\Util\RSAKey), '\xAA\x80"\x8DBAuO\xD6B\xF6f\xA7\x06\xAC...', 'sha256') 
#1 /path-to-www/vendor/web-token/jwt-encryption-algorithm-rsa/RSA.php(34): Jose\Component\Encryption\Algorithm\KeyEncryption\Util\RSACrypt::encrypt(Object(Jose\Component\Core\Util\RSAKey), '\xAA\x80"\x8DBAuO\xD6B\xF6f\xA7\x06\xAC...', 1, 'sha256') 
#2 /path-to-www/vendor/web-token/jwt-encryption/JWEBuilder.php(390): Jos in /path-to-www/vendor/web-token/jwt-encryption-algorithm-rsa/Util/RSACrypt.php on line 123
```
